### PR TITLE
202 when to take your pensions

### PR DESCRIPTION
**Pull request by ZhuZhuThanou**
### Description
Added the When to Take Your Government Pensions learning page as defined by the HiFi working designs

List of proposed changes:

- add the When to Take Your Government Pensions learning page
- add a link for the index learning page
- provided both content json files for EN/FR

### What to test for/How to test
npm i
npm run dev
Open browser to 
http://localhost:3000/fr/learn/when-to-take-your-government-pensions
http://localhost:3000/en/learn/when-to-take-your-government-pensions

### Additional Notes
The work is done to put the UI structure in place. Finalized text will come and be worked on in a different story.

### CODE REVIEW

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx
Comment (gregory-j-baker) [inline frontend/src/pages/learn/when-to-take-your-pensions.tsx:1]: Many of the images on this page are not a11y accessible.

Complex charts and graphs should be accompanied by both a text summary and a data table.

**References:**

- https://design.canada.ca/common-design-patterns/charts-graphs.html

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+              col1: t('old-age-security.table-one.row-one.column-one-value'),
+              col2: t('old-age-security.table-one.row-one.column-two-value'),
+              col3: t('old-age-security.table-one.row-one.column-three-value')
+            },
+            {
+              col1: t('old-age-security.table-one.row-two.column-one-value'),
+              col2: t('old-age-security.table-one.row-two.column-two-value'),
+              col3: t('old-age-security.table-one.row-two.column-three-value')
+            }].map((row, index) => (
+              <tr key={index} className={`${index === 0 ? 'bg-[#B9B9B9]' : ''}`}>
```
Comment (gregory-j-baker) [inline frontend/src/pages/learn/when-to-take-your-pensions.tsx:82]: Indexes should not be used as keys.

**References:**

- https://react.dev/learn/rendering-lists

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+                href={t(
+                  'old-age-security.smart-tip.link-two'
+                )}
+              />
+            )
+          }}
+        />
+      </ImportantCard>
+      <p>{t('old-age-security.content-two')}</p>
+      <div className="w-full lg:flex-1">
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/when-to-take-your-pensions.tsx:114]: A block-level element is an HTML element that takes up horizontal space by default. It stretches 100% wide and takes on the height of its inner content. Do the same for other divs with same classes.
```suggestion
      <div>
```

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx
Comment (ZhuZhuThanou) [reply frontend/src/pages/learn/when-to-take-your-pensions.tsx:1]: Designer have not provided the text summary and data table. Okay to put placeholders?

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx (hunk 1 above)
Comment (ZhuZhuThanou) [reply frontend/src/pages/learn/when-to-take-your-pensions.tsx:82]: add id to each table row and not relying on array index

File: frontend/public/assets/cpp-income-table-en.svg
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/public/assets/cpp-income-table-en.svg:1]: converted into a table

File: frontend/src/pages/learn/when-to-take-your-pensions.tsx
Comment (ZhuZhuThanou) [reply frontend/src/pages/learn/when-to-take-your-pensions.tsx:1]: Added a component to hold additional contents and data for accessibility

**Pull request merged**